modules/build/src/test/scala/scala/build/tests/SourcesTests.scala

```diff
@@ -69,44 +69,46 @@ class SourcesTests extends munit.FunSuite {
       val scopedSources = crossSources.scopedSources(BuildOptions()).orThrow
       val sources       = scopedSources.sources(Scope.Main, BuildOptions())
 
-      expect(sources.buildOptions.classPathOptions.extraDependencies.map(_.value) == expectedDeps)
+      expect(sources.buildOptions.classPathOptions.extraDependencies.map(
+        _.value
+      ).toSet == expectedDeps.toSet)
```

Comment:
Why is this changed to a set? The order of dependencies matters (it has an impact on the order of JARs in the class path, which can make the resources or classes of certain JARs shadow the same resources or classes in other JARs).

Reply:
You're right. Changed that